shared-libs/cht-datasource/src/person.ts

```diff
@@ -44,6 +44,18 @@ export namespace v1 {
       };
     };
 
+  const getPeople = <T>(
+    localFn: (c: LocalDataContext) => (limit: number, skip: number) => Promise<T>,
+    remoteFn: (c: RemoteDataContext) => (limit: number, skip: number) => Promise<T>
+  ) => (context: DataContext) => {
+      assertDataContext(context);
+      const fn = adapt(context, localFn, remoteFn);
+
+      return async (limit = 100, skip = 0): Promise<T> => {
```

Comment:
sure, let's do that

Reply:
As you mentioned above, `type` is a keyword, so maybe `contactType` would be a better property name.

Reply:
Now that I think about this, maybe the type could also be named `ContactTypeQualifier` since we're explicitly dealing with `Contact Types` with this qualifier.